tests/unit/components/LemmaInput.test.ts

```diff
@@ -57,6 +57,17 @@ describe( 'LemmaInput', () => {
 
 			expect( lemmaInputWrapper.get( '.wikit-ValidationMessage--error' ).text() ).toContain( '⧼wikibaselexeme-newlexeme-error-no-lemma⧽' );
 		} );
+
+		it( ':error - displays an error message in case input is too long', async () => {
+			const tooLongValue =
```

Comment:
This does indeed work for basically any value here, because you're not setting `config.maxLemmaLength` anywhere. Have a look at the `createComponent(...)` helper function in this class. It allows you to override part of the default configuration. You can do that with a smaller value than 1000, then it is easier to test.

Also, this should include a second test case that makes sure you're counting code-points, not double-bytes. That is, "`🏳️‍🌈`" should be counted as length 4, not length 6.